<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>TestNG Report</title><style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}</style></head><body><table><tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr><tr><th colspan="7">CalculatorTestSuite</th></tr><tr><td><a href="#t0">CalculatorTestSub</a></td><td class="num">14</td><td class="num">0</td><td class="num">0</td><td class="num">115</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t1">CalculatorTestDiv</a></td><td class="num">18</td><td class="num">0</td><td class="num attn">5</td><td class="num">157</td><td></td><td></td></tr><tr><td><a href="#t2">CalculatorTestSum</a></td><td class="num">14</td><td class="num">0</td><td class="num">0</td><td class="num">85</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t3">CalculatorTestMult</a></td><td class="num">7</td><td class="num">0</td><td class="num attn">5</td><td class="num">105</td><td></td><td></td></tr><tr><td><a href="#t4">CalculatorTestTg</a></td><td class="num">0</td><td class="num">0</td><td class="num attn">7</td><td class="num">13</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t5">CalculatorTestCtg</a></td><td class="num">0</td><td class="num">0</td><td class="num attn">7</td><td class="num">17</td><td></td><td></td></tr><tr><td><a href="#t6">CalculatorTestPow</a></td><td class="num">7</td><td class="num">0</td><td class="num">0</td><td class="num">122</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t7">CalculatorTestSqrt</a></td><td class="num">5</td><td class="num">0</td><td class="num attn">1</td><td class="num">10</td><td></td><td></td></tr><tr><td><a href="#t8">CalculatorTestCos</a></td><td class="num">0</td><td class="num">0</td><td class="num attn">7</td><td class="num">42</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t9">CalculatorTestIsPositive</a></td><td class="num">5</td><td class="num">0</td><td class="num">0</td><td class="num">33</td><td></td><td></td></tr><tr><td><a href="#t10">CalculatorTestSin</a></td><td class="num">7</td><td class="num">0</td><td class="num">0</td><td class="num">6</td><td></td><td></td></tr><tr class="stripe"><td><a href="#t11">CalculatorTestIsNegative</a></td><td class="num">5</td><td class="num">0</td><td class="num">0</td><td class="num">33</td><td></td><td></td></tr><tr><th>Total</th><th class="num">82</th><th class="num">0</th><th class="num attn">32</th><th class="num">738</th><th colspan="2"></th></tr></table><table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">CalculatorTestSuite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">CalculatorTestSub &#8212; passed</th></tr><tr class="passedeven"><td rowspan="14">by.epam.atmentoring.unittest.CalculatorTestSub</td><td><a href="#m0">testSub</a></td><td rowspan="14">1497173052271</td><td rowspan="14">2</td></tr><tr class="passedeven"><td><a href="#m1">testSub</a></td></tr><tr class="passedeven"><td><a href="#m2">testSub</a></td></tr><tr class="passedeven"><td><a href="#m3">testSub</a></td></tr><tr class="passedeven"><td><a href="#m4">testSub</a></td></tr><tr class="passedeven"><td><a href="#m5">testSub</a></td></tr><tr class="passedeven"><td><a href="#m6">testSub</a></td></tr><tr class="passedeven"><td><a href="#m7">testSub</a></td></tr><tr class="passedeven"><td><a href="#m8">testSub</a></td></tr><tr class="passedeven"><td><a href="#m9">testSub</a></td></tr><tr class="passedeven"><td><a href="#m10">testSub</a></td></tr><tr class="passedeven"><td><a href="#m11">testSub</a></td></tr><tr class="passedeven"><td><a href="#m12">testSub</a></td></tr><tr class="passedeven"><td><a href="#m13">testSub</a></td></tr></tbody><tbody id="t1"><tr><th colspan="4">CalculatorTestDiv &#8212; failed</th></tr><tr class="failedeven"><td rowspan="5">by.epam.atmentoring.unittest.CalculatorTestDiv</td><td><a href="#m14">testDiv</a></td><td rowspan="5">1497173052220</td><td rowspan="5">0</td></tr><tr class="failedeven"><td><a href="#m15">testDiv</a></td></tr><tr class="failedeven"><td><a href="#m16">testDiv</a></td></tr><tr class="failedeven"><td><a href="#m17">testDiv</a></td></tr><tr class="failedeven"><td><a href="#m18">testDiv</a></td></tr><tr><th colspan="4">CalculatorTestDiv &#8212; passed</th></tr><tr class="passedeven"><td rowspan="18">by.epam.atmentoring.unittest.CalculatorTestDiv</td><td><a href="#m19">testDiv</a></td><td rowspan="18">1497173052317</td><td rowspan="18">1</td></tr><tr class="passedeven"><td><a href="#m20">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m21">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m22">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m23">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m24">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m25">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m26">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m27">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m28">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m29">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m30">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m31">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m32">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m33">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m34">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m35">testDiv</a></td></tr><tr class="passedeven"><td><a href="#m36">testDiv</a></td></tr></tbody><tbody id="t2"><tr><th colspan="4">CalculatorTestSum &#8212; passed</th></tr><tr class="passedeven"><td rowspan="14">by.epam.atmentoring.unittest.CalculatorTestSum</td><td><a href="#m37">testSum</a></td><td rowspan="14">1497173052253</td><td rowspan="14">0</td></tr><tr class="passedeven"><td><a href="#m38">testSum</a></td></tr><tr class="passedeven"><td><a href="#m39">testSum</a></td></tr><tr class="passedeven"><td><a href="#m40">testSum</a></td></tr><tr class="passedeven"><td><a href="#m41">testSum</a></td></tr><tr class="passedeven"><td><a href="#m42">testSum</a></td></tr><tr class="passedeven"><td><a href="#m43">testSum</a></td></tr><tr class="passedeven"><td><a href="#m44">testSum</a></td></tr><tr class="passedeven"><td><a href="#m45">testSum</a></td></tr><tr class="passedeven"><td><a href="#m46">testSum</a></td></tr><tr class="passedeven"><td><a href="#m47">testSum</a></td></tr><tr class="passedeven"><td><a href="#m48">testSum</a></td></tr><tr class="passedeven"><td><a href="#m49">testSum</a></td></tr><tr class="passedeven"><td><a href="#m50">testSum</a></td></tr></tbody><tbody id="t3"><tr><th colspan="4">CalculatorTestMult &#8212; failed</th></tr><tr class="failedeven"><td rowspan="5">by.epam.atmentoring.unittest.CalculatorTestMult</td><td><a href="#m51">testMult</a></td><td rowspan="5">1497173052213</td><td rowspan="5">2</td></tr><tr class="failedeven"><td><a href="#m52">testMult</a></td></tr><tr class="failedeven"><td><a href="#m53">testMult</a></td></tr><tr class="failedeven"><td><a href="#m54">testMult</a></td></tr><tr class="failedeven"><td><a href="#m55">testMult</a></td></tr><tr><th colspan="4">CalculatorTestMult &#8212; passed</th></tr><tr class="passedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestMult</td><td><a href="#m56">testMult</a></td><td rowspan="7">1497173052235</td><td rowspan="7">1</td></tr><tr class="passedeven"><td><a href="#m57">testMult</a></td></tr><tr class="passedeven"><td><a href="#m58">testMult</a></td></tr><tr class="passedeven"><td><a href="#m59">testMult</a></td></tr><tr class="passedeven"><td><a href="#m60">testMult</a></td></tr><tr class="passedeven"><td><a href="#m61">testMult</a></td></tr><tr class="passedeven"><td><a href="#m62">testMult</a></td></tr></tbody><tbody id="t4"><tr><th colspan="4">CalculatorTestTg &#8212; failed</th></tr><tr class="failedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestTg</td><td><a href="#m63">testTg</a></td><td rowspan="7">1497173052438</td><td rowspan="7">1</td></tr><tr class="failedeven"><td><a href="#m64">testTg</a></td></tr><tr class="failedeven"><td><a href="#m65">testTg</a></td></tr><tr class="failedeven"><td><a href="#m66">testTg</a></td></tr><tr class="failedeven"><td><a href="#m67">testTg</a></td></tr><tr class="failedeven"><td><a href="#m68">testTg</a></td></tr><tr class="failedeven"><td><a href="#m69">testTg</a></td></tr></tbody><tbody id="t5"><tr><th colspan="4">CalculatorTestCtg &#8212; failed</th></tr><tr class="failedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestCtg</td><td><a href="#m70">testCtg</a></td><td rowspan="7">1497173052448</td><td rowspan="7">1</td></tr><tr class="failedeven"><td><a href="#m71">testCtg</a></td></tr><tr class="failedeven"><td><a href="#m72">testCtg</a></td></tr><tr class="failedeven"><td><a href="#m73">testCtg</a></td></tr><tr class="failedeven"><td><a href="#m74">testCtg</a></td></tr><tr class="failedeven"><td><a href="#m75">testCtg</a></td></tr><tr class="failedeven"><td><a href="#m76">testCtg</a></td></tr></tbody><tbody id="t6"><tr><th colspan="4">CalculatorTestPow &#8212; passed</th></tr><tr class="passedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestPow</td><td><a href="#m77">testPow</a></td><td rowspan="7">1497173052390</td><td rowspan="7">10</td></tr><tr class="passedeven"><td><a href="#m78">testPow</a></td></tr><tr class="passedeven"><td><a href="#m79">testPow</a></td></tr><tr class="passedeven"><td><a href="#m80">testPow</a></td></tr><tr class="passedeven"><td><a href="#m81">testPow</a></td></tr><tr class="passedeven"><td><a href="#m82">testPow</a></td></tr><tr class="passedeven"><td><a href="#m83">testPow</a></td></tr></tbody><tbody id="t7"><tr><th colspan="4">CalculatorTestSqrt &#8212; failed</th></tr><tr class="failedeven"><td rowspan="1">by.epam.atmentoring.unittest.CalculatorTestSqrt</td><td><a href="#m84">testSqrt</a></td><td rowspan="1">1497173052432</td><td rowspan="1">1</td></tr><tr><th colspan="4">CalculatorTestSqrt &#8212; passed</th></tr><tr class="passedeven"><td rowspan="5">by.epam.atmentoring.unittest.CalculatorTestSqrt</td><td><a href="#m85">testSqrt</a></td><td rowspan="5">1497173052438</td><td rowspan="5">0</td></tr><tr class="passedeven"><td><a href="#m86">testSqrt</a></td></tr><tr class="passedeven"><td><a href="#m87">testSqrt</a></td></tr><tr class="passedeven"><td><a href="#m88">testSqrt</a></td></tr><tr class="passedeven"><td><a href="#m89">testSqrt</a></td></tr></tbody><tbody id="t8"><tr><th colspan="4">CalculatorTestCos &#8212; failed</th></tr><tr class="failedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestCos</td><td><a href="#m90">testCos</a></td><td rowspan="7">1497173052494</td><td rowspan="7">1</td></tr><tr class="failedeven"><td><a href="#m91">testCos</a></td></tr><tr class="failedeven"><td><a href="#m92">testCos</a></td></tr><tr class="failedeven"><td><a href="#m93">testCos</a></td></tr><tr class="failedeven"><td><a href="#m94">testCos</a></td></tr><tr class="failedeven"><td><a href="#m95">testCos</a></td></tr><tr class="failedeven"><td><a href="#m96">testCos</a></td></tr></tbody><tbody id="t9"><tr><th colspan="4">CalculatorTestIsPositive &#8212; passed</th></tr><tr class="passedeven"><td rowspan="5">by.epam.atmentoring.unittest.CalculatorTestIsPositive</td><td><a href="#m97">testIsPositive</a></td><td rowspan="5">1497173052519</td><td rowspan="5">7</td></tr><tr class="passedeven"><td><a href="#m98">testIsPositive</a></td></tr><tr class="passedeven"><td><a href="#m99">testIsPositive</a></td></tr><tr class="passedeven"><td><a href="#m100">testIsPositive</a></td></tr><tr class="passedeven"><td><a href="#m101">testIsPositive</a></td></tr></tbody><tbody id="t10"><tr><th colspan="4">CalculatorTestSin &#8212; passed</th></tr><tr class="passedeven"><td rowspan="7">by.epam.atmentoring.unittest.CalculatorTestSin</td><td><a href="#m102">testSin</a></td><td rowspan="7">1497173052472</td><td rowspan="7">0</td></tr><tr class="passedeven"><td><a href="#m103">testSin</a></td></tr><tr class="passedeven"><td><a href="#m104">testSin</a></td></tr><tr class="passedeven"><td><a href="#m105">testSin</a></td></tr><tr class="passedeven"><td><a href="#m106">testSin</a></td></tr><tr class="passedeven"><td><a href="#m107">testSin</a></td></tr><tr class="passedeven"><td><a href="#m108">testSin</a></td></tr></tbody><tbody id="t11"><tr><th colspan="4">CalculatorTestIsNegative &#8212; passed</th></tr><tr class="passedeven"><td rowspan="5">by.epam.atmentoring.unittest.CalculatorTestIsNegative</td><td><a href="#m109">testIsNegative</a></td><td rowspan="5">1497173052558</td><td rowspan="5">0</td></tr><tr class="passedeven"><td><a href="#m110">testIsNegative</a></td></tr><tr class="passedeven"><td><a href="#m111">testIsNegative</a></td></tr><tr class="passedeven"><td><a href="#m112">testIsNegative</a></td></tr><tr class="passedeven"><td><a href="#m113">testIsNegative</a></td></tr></tbody></table><h2>CalculatorTestSub</h2><h3 id="m0">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-9223372036854775808</td><td>-1</td><td>-9223372036854775807</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m1">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.5</td><td>1.5</td><td>-3.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m2">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>0</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m3">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0.1</td><td>1.01</td><td>-0.91</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m4">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>9223372036854775807</td><td>1</td><td>9223372036854775806</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m5">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>-1</td><td>1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m6">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-3</td><td>-2</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m7">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>1</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m8">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.7976931348623157E308</td><td>-1</td><td>-1.7976931348623157E308</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m9">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.7976931348623157E308</td><td>1</td><td>1.7976931348623157E308</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m10">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-2</td><td>3</td><td>-5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m11">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2</td><td>3</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m12">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-3</td><td>2</td><td>-5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m13">by.epam.atmentoring.unittest.CalculatorTestSub#testSub</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.5</td><td>1.5</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestDiv</h2><h3 id="m14">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>1.7976931348623157E308</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestDiv.testDiv(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestDiv@6767c1fc] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m15">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>-1.7976931348623157E308</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestDiv.testDiv(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestDiv@6767c1fc] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m16">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>0.0</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestDiv.testDiv(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestDiv@6767c1fc] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m17">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>-1.0</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestDiv.testDiv(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestDiv@6767c1fc] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m18">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>1.0</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestDiv.testDiv(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestDiv@6767c1fc] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m19">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1</td><td>3</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m20">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>9223372036854775807</td></tr><tr><th>Expected Exception</th></tr><tr><td><div class="stacktrace">java.lang.NumberFormatException: Attempt to divide by zero
	at com.epam.tat.module4.Calculator.div(Calculator.java:34)
	at by.epam.atmentoring.unittest.CalculatorTestDiv.testDiv(CalculatorTestDiv.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m21">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.1</td><td>0.1</td><td>11</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m22">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>0</td></tr><tr><th>Expected Exception</th></tr><tr><td><div class="stacktrace">java.lang.NumberFormatException: Attempt to divide by zero
	at com.epam.tat.module4.Calculator.div(Calculator.java:34)
	at by.epam.atmentoring.unittest.CalculatorTestDiv.testDiv(CalculatorTestDiv.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m23">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0.0</td><td>123.45678</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m24">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-9223372036854775808</td><td>2</td><td>-4611686018427387904</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m25">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>10</td><td>3</td><td>3</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m26">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>10</td><td>3</td><td>3.3333333333333335</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m27">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>225</td><td>15</td><td>15</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m28">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>-9223372036854775808</td></tr><tr><th>Expected Exception</th></tr><tr><td><div class="stacktrace">java.lang.NumberFormatException: Attempt to divide by zero
	at com.epam.tat.module4.Calculator.div(Calculator.java:34)
	at by.epam.atmentoring.unittest.CalculatorTestDiv.testDiv(CalculatorTestDiv.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m29">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>1</td></tr><tr><th>Expected Exception</th></tr><tr><td><div class="stacktrace">java.lang.NumberFormatException: Attempt to divide by zero
	at com.epam.tat.module4.Calculator.div(Calculator.java:34)
	at by.epam.atmentoring.unittest.CalculatorTestDiv.testDiv(CalculatorTestDiv.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m30">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>10</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m31">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.7976931348623157E308</td><td>2</td><td>8.988465674311579E307</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m32">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.7976931348623157E308</td><td>-2</td><td>8.988465674311579E307</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m33">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>-1</td></tr><tr><th>Expected Exception</th></tr><tr><td><div class="stacktrace">java.lang.NumberFormatException: Attempt to divide by zero
	at com.epam.tat.module4.Calculator.div(Calculator.java:34)
	at by.epam.atmentoring.unittest.CalculatorTestDiv.testDiv(CalculatorTestDiv.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m34">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>9223372036854775807</td><td>2</td><td>4611686018427387903</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m35">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-5.1</td><td>2.0</td><td>-2.55</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m36">by.epam.atmentoring.unittest.CalculatorTestDiv#testDiv</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-42</td><td>-7</td><td>6</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestSum</h2><h3 id="m37">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>-1</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m38">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>0</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m39">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.7976931348623157E308</td><td>-1</td><td>1.7976931348623157E308</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m40">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-3</td><td>-2</td><td>-5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m41">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-2</td><td>3</td><td>1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m42">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>9223372036854775807</td><td>-1</td><td>9223372036854775806</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m43">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-3</td><td>2</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m44">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.7976931348623157E308</td><td>1</td><td>-1.7976931348623157E308</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m45">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.5</td><td>1.5</td><td>3.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m46">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0.1</td><td>1.01</td><td>1.11</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m47">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-9223372036854775808</td><td>1</td><td>-9223372036854775807</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m48">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0</td><td>1</td><td>1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m49">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2</td><td>3</td><td>5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m50">by.epam.atmentoring.unittest.CalculatorTestSum#testSum</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.5</td><td>1.5</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestMult</h2><h3 id="m51">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.5</td><td>0.0</td><td>0.0</td></tr><tr><th colspan="3">Exception</th></tr><tr><td colspan="3"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [-0.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestMult.testMult(CalculatorTestMult.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m52">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.5</td><td>0.75</td><td>1.125</td></tr><tr><th colspan="3">Exception</th></tr><tr><td colspan="3"><div class="stacktrace">java.lang.AssertionError: expected [1.125] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestMult.testMult(CalculatorTestMult.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m53">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.5</td><td>0.75</td><td>-1.125</td></tr><tr><th colspan="3">Exception</th></tr><tr><td colspan="3"><div class="stacktrace">java.lang.AssertionError: expected [-1.125] but found [-2.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestMult.testMult(CalculatorTestMult.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m54">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1.5</td><td>-1.5</td><td>2.25</td></tr><tr><th colspan="3">Exception</th></tr><tr><td colspan="3"><div class="stacktrace">java.lang.AssertionError: expected [2.25] but found [2.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestMult.testMult(CalculatorTestMult.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m55">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.5</td><td>1.5</td><td>2.25</td></tr><tr><th colspan="3">Exception</th></tr><tr><td colspan="3"><div class="stacktrace">java.lang.AssertionError: expected [2.25] but found [2.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestMult.testMult(CalculatorTestMult.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m56">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1</td><td>0</td><td>0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m57">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1</td><td>1</td><td>-1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m58">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1</td><td>1</td><td>1</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m59">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-1</td><td>10</td><td>-10</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m60">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1.5</td><td>0.0</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m61">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>0.0</td><td>1.5</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m62">by.epam.atmentoring.unittest.CalculatorTestMult#testMult</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>1</td><td>10</td><td>10</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestTg</h2><h3 id="m63">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0.0175</td><td>0.0175</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0175] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m64">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-6.2832</td><td>0.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m65">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.5708</td><td>Infinity</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [Infinity] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m66">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>3.1416</td><td>0.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m67">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>4.7997</td><td>-11.4301</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [-11.4301] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m68">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.0472</td><td>1.7321</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [1.7321] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m69">by.epam.atmentoring.unittest.CalculatorTestTg#testTg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>6.2832</td><td>0.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestTg.testTg(CalculatorTestTg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestCtg</h2><h3 id="m70">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0.0175</td><td>57.29</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [57.29] but found [0.01749821376048069]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m71">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.5708</td><td>0.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [0.9171529190863275]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m72">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>6.2832</td><td>Infinity</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [Infinity] but found [0.9999930255445617]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m73">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>3.1416</td><td>-Infinity</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [-Infinity] but found [0.9962721308919683]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m74">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>4.7997</td><td>-0.0875</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [-0.0875] but found [0.9998644704129042]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m75">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.0472</td><td>0.5774</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.5774] but found [0.7807153915783637]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m76">by.epam.atmentoring.unittest.CalculatorTestCtg#testCtg</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-6.2832</td><td>Infinity</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [Infinity] but found [-0.9999930255445617]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCtg.testCtg(CalculatorTestCtg.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestPow</h2><h3 id="m77">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-2.0</td><td>2.0</td><td>4.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m78">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-2.0</td><td>1.0</td><td>-2.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m79">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>-2.0</td><td>0.0</td><td>1.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m80">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2.0</td><td>1.0</td><td>2.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m81">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2.0</td><td>2.0</td><td>4.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m82">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2.0</td><td>-1.0</td><td>0.5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m83">by.epam.atmentoring.unittest.CalculatorTestPow#testPow</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th><th>Parameter #3</th></tr><tr class="param stripe"><td>2.0</td><td>0.0</td><td>1.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestSqrt</h2><h3 id="m84">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>-1.0</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestException: 
Method CalculatorTestSqrt.testSqrt(double)[pri:0, instance:by.epam.atmentoring.unittest.CalculatorTestSqrt@46d56d67] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m85">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>10.0</td><td>3.1622776601683795</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m86">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>2.25</td><td>1.5</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m87">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>225.0</td><td>15.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m88">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.0</td><td>1.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m89">by.epam.atmentoring.unittest.CalculatorTestSqrt#testSqrt</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0.0</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestCos</h2><h3 id="m90">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>3.1416</td><td>-1.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [-1.0] but found [0.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m91">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>4.7997</td><td>0.0872</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0872] but found [-0.9962]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m92">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.0472</td><td>0.5</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.5] but found [0.866]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m93">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.5708</td><td>0.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.0] but found [1.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m94">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>6.2832</td><td>1.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [1.0] but found [0.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m95">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-6.2832</td><td>-1.0</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [-1.0] but found [0.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m96">by.epam.atmentoring.unittest.CalculatorTestCos#testCos</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0.0175</td><td>0.9998</td></tr><tr><th colspan="2">Exception</th></tr><tr><td colspan="2"><div class="stacktrace">java.lang.AssertionError: expected [0.9998] but found [0.0175]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at by.epam.atmentoring.unittest.CalculatorTestCos.testCos(CalculatorTestCos.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestIsPositive</h2><h3 id="m97">by.epam.atmentoring.unittest.CalculatorTestIsPositive#testIsPositive</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-9223372036854775808</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m98">by.epam.atmentoring.unittest.CalculatorTestIsPositive#testIsPositive</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>9223372036854775807</td><td>true</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m99">by.epam.atmentoring.unittest.CalculatorTestIsPositive#testIsPositive</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-1</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m100">by.epam.atmentoring.unittest.CalculatorTestIsPositive#testIsPositive</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m101">by.epam.atmentoring.unittest.CalculatorTestIsPositive#testIsPositive</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1</td><td>true</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestSin</h2><h3 id="m102">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>6.2832</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m103">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>4.7997</td><td>-0.9962</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m104">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0.0175</td><td>0.0175</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m105">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.0472</td><td>0.866</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m106">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1.5708</td><td>1.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m107">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>3.1416</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m108">by.epam.atmentoring.unittest.CalculatorTestSin#testSin</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-6.2832</td><td>0.0</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h2>CalculatorTestIsNegative</h2><h3 id="m109">by.epam.atmentoring.unittest.CalculatorTestIsNegative#testIsNegative</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-1</td><td>true</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m110">by.epam.atmentoring.unittest.CalculatorTestIsNegative#testIsNegative</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>1</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m111">by.epam.atmentoring.unittest.CalculatorTestIsNegative#testIsNegative</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>0</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m112">by.epam.atmentoring.unittest.CalculatorTestIsNegative#testIsNegative</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>-9223372036854775808</td><td>true</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m113">by.epam.atmentoring.unittest.CalculatorTestIsNegative#testIsNegative</h3><table class="result"><tr class="param"><th>Parameter #1</th><th>Parameter #2</th></tr><tr class="param stripe"><td>9223372036854775807</td><td>false</td></tr></table><p class="totop"><a href="#summary">back to summary</a></p></body></html>